pkgs/applications/virtualization/virt-manager/default.nix

```diff
@@ -64,6 +64,10 @@ python3.pkgs.buildPythonApplication rec {
     gappsWrapperArgs+=(--prefix PATH : "${lib.makeBinPath [ cpio e2fsprogs file findutils gzip ]}")
 
     makeWrapperArgs+=("''${gappsWrapperArgs[@]}")
+
+    # Fixes testCLI0051virt_install_initrd_inject on Darwin: "cpio: root:root: invalid group"
+    substituteInPlace virtinst/install/installerinject.py \
+      --replace "'--owner=root:root'" "'--owner=0:0'"
```

Comment:
Sounds like this patch should be sent upstream.

Reply:
virt-manager/virt-manager#423.

Reply:
@wegank: Thank you!